backlog.md

```diff
@@ -0,0 +1,16 @@
+# Developer backlog
+
+## Main
+
+- Handle Ctrl+C signal so it can exit with 0 instead of non-zero.
+
+
+## Request parsing
+
+RFC7230 Section 3.1.1
+
+- Should allow a request line of at least 8,000 octets.  On the flip side, prevent a
+  Denial of Service attack with a request line that (almost) never ends (has no CR).
+  In this case, it should only read up to 8,000 octets and give up if it hasn't seen CR yet.
```

Comment:
This is a really cool idea. If you get it worked out, it could maybe be the sort of thing you add a test for in CobSpec. That said, I also wouldn't let it get in the way of writing code to pass the existing tests.

Reply:
Thanks, Rob.  I'll put it down for some possible Waza time, later.